portal/views/portal.py

```diff
@@ -594,12 +594,21 @@ def initial_queries():
             r = ROLE.STAFF.value if r == ROLE.STAFF_ADMIN.value else r
             role = r
     terms = get_terms(user.locale_code, org, role)
+    substudy_terms = None
+    #TODO check if this is OK?
```

Comment:
Not clear on your TODO - but in general, all imports should go at the head of the file, UNLESS that creates an include cycle preventing the app from loading.